PG.StarWarsGame.Files.MEG/PG.StarWarsGame.Files.MEG/Binary/File/Type/Definition/V1/MegFileNameTable.cs

```diff
@@ -6,17 +9,17 @@
 
 [assembly: InternalsVisibleTo("PG.StarWarsGame.Files.MEGs.Test")]
 
-namespace PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition
+namespace PG.StarWarsGame.Files.MEG.Binary.File.Type.Definition.V1
 {
     internal class MegFileNameTable : IBinaryFile, ISizeable
```

Comment:
Would it make sense to make the type implement `IEnumerable<MetFileNameTableRecord>` ?